habitatstreams/src/22_progress_evaluation_3260.Rmd

```diff
@@ -0,0 +1,522 @@
+  ungroup()
+  
+```
+
+
+```{r figpunten, fig.width= 7}
+
+sample_units_to_do %>%
+  st_transform(crs = 4326) %>%
+  mutate(color_marker = ifelse(in_map3260, "blue", "red")) %>%
+  leaflet() %>%
+  addTiles() %>%
+  addCircleMarkers(color = ~color_marker, label = ~sampling_unit_code) %>%
+  addPolylines(data = st_transform(map_3260_new, crs = 4326))
+  
+```
+
+
+## Output standaardiseren
+
+We standaardiseren het overzicht van de steekproefeenheden en de opmetingen op een analoge manier als bij de terrestrische meetpunten. We maken daarvoor gebruik van onderstaande tabellen.
+
+```{r}
+sample_assessments_3260_2 <- sample_assessments_3260 %>%
+  filter(!is.na(klasse) | sampling_unit_code %in% sample_units_to_do$sampling_unit_code) %>%
+  mutate(grts_ranking_draw =grts_ranking,
+         type_target = "3260",
+         scheme = "MHQ3260",
+         distance = round(sqrt((x - x_assessment)^2 + (y - y_assessment)^2), 0)) %>%
+  group_by(grts_ranking_draw) %>%
+  mutate(point_code = str_c(grts_ranking_draw, "_", rank(x,y))) %>%
+  ungroup() %>%
+  left_join(sample_rapport, by = c("grts_ranking", "sampling_unit_code"))
+
+
+sample_assessments_3260_replaced <- sample_assessments_3260_2 %>%
+  filter(position_change) %>%
+  mutate(x = x_assessment,
+         y = y_assessment) %>%
+  st_as_sf(coords = c("x_assessment", "y_assessment"), crs = 31370) %>%
+  mutate(grts_ranking = grts_master[as(., "Spatial")],
+         point_code = str_c(grts_ranking_draw, "_2")) %>%
+  st_drop_geometry()
+
+
+sample_assessments_3260_original <- sample_assessments_3260_2 %>%
+  mutate(field_code = ifelse(position_change, NA, field_code),
+         segment_measurable = ifelse(position_change, FALSE, segment_measurable),
+         type_observed = ifelse(position_change, NA, type_observed),
+         is_type_target = ifelse(position_change, NA, is_type_target),
+         position_change = FALSE,
+         distance = 0)
+  
+sample_assessments_3260_update <- sample_assessments_3260_original %>%
+  bind_rows(sample_assessments_3260_replaced) %>%
+  mutate(segment_id = sampling_unit_code,
+         sampling_unit_code = str_c(grts_ranking, "_3260"),
+         x = round(x, 2),
+         y = round(y, 2),
+         sac_streams = factor(sac_streams))
+       
+```
+
+
+### mhq_streams_refpoints
+
+De tabel 'mhq_streams_refpoints' geeft een overzicht van de zogenaamde referentiepunten voor alle steekproefeenheden in het meetnet voor habitattype 3260. Elke steekproefeenheid heeft een referentiepunt waar naar toe genavigeerd wordt om de meting uit te voeren. Bij stromende wateren is het referentiepunt het startpunt van het 100 meter segment (en meer specifiek het meest stroomafwaarts gelegen punt van het segment).
+
+De tabel bestaat uit volgende variabelen:
+
++ scheme: code voor het meetnet;
++ sampling_unit_code: code voor de steekproefeenheid;
++ segment_id: id van het riviersegment;
++ point_code: code voor het referentiepunt;
```

Comment:
I suggest to drop one of these three. Why have different codes for `segment_id` and `sampling_unit_code`? Sampling units are the segments, a distinction between segments and (linked) points is the only thing you need IMO. Also, in the text  best state more clearly what defines unique rows (key).

Reply:
eventually we came to the conclusion that the connection between point_code and segment_id (or lake_id) is better put in a separate table. It is a difference between terrestrial and aquatic; so it would make the tables conform to the terrestrial format